.github/actions/javascript/reviewerChecklist/index.js

```diff
@@ -30,6 +31,7 @@ function getNumberOfItemsFromReviewerChecklist() {
             });
             res.on('end', () => {
                 const numberOfChecklistItems = (fileContents.match(/- \[ \]/g) || []).length;
+                console.log(`There are ${numberOfChecklistItems} items in the reviewer checklist.`);
```

Comment:
Can you please also add a log inside the `.on('error', reject);` block too? Just in case for some reason the promise isn't being resolved correctly?